src/App.tsx

```diff
@@ -1,22 +1,23 @@
 import { useState } from "react";
 import Button from "@mui/material/Button";
 import Login from "./components/Login";
-import { Header } from "./components/Header/header";
+// import { Header } from "./components/Header/Header";
 import { HomePage } from "./components/HomePage/homePage";
 import { LoginForm } from "./components/LoginForm/loginForm";
-import { NavLinkBar } from "./components/Header/navLinkBar";
-import { ProfileModal } from "./components/Header/profileModal";
+import { NavLinkMenu } from "./components/Header/NavLinkMenu";
+// import { ProfileModal } from "./components/Header/profileModal";
 import {
   headerIcons,
   headerLinks,
   navLinks,
-} from "./components/Header/headerDetails";
+} from "./components/Header/HeaderDetails";
+import { Header } from "./components/Header/Header";
 
 function App() {
   return (
     <div className="App">
       <Header headerLinks={headerLinks} headerIcons={headerIcons} />
```

Comment:
since the header links are not used by other components. They should be imported directly into the Header component